### PR TITLE
Add selenium test to make sure platforms appear in search results ...

### PR DESCRIPTION
**Pull request by akashisama**
...after being selected in facets

### CODE REVIEW

**Comment by akashisama** [comment]
The test fails currently because it's catching bug #2506

File: selenium/src/test/java/au/org/emii/portal/tests/step1/FacetPlatformTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        for (WebElement result : results) {
+            try {
+                WebElement facetResult = result.findElement(By.xpath("div[4]/span[2]"));
+                Assert.assertTrue(facetResult.getText().contains(parameter));
+            } catch (NoSuchElementException e) {
+                Assert.fail("No platform section in results but " + parameter + " parameter should be present");
+            }
+
+        }
+
```
Comment (anguss00) [inline selenium/src/test/java/au/org/emii/portal/tests/step1/FacetPlatformTest.java]: Sorry to be mr pedantic... extra spaces here

File: selenium/src/test/java/au/org/emii/portal/tests/step1/FacetPlatformTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.List;
+
+public class FacetPlatformTest extends BaseTest {
+
+    private static Logger log = Logger.getLogger(FacetPlatformTest.class.getName());
+
+    @Test
+    public void platformTest() throws InterruptedException {
+        getDriver().get(AODN_PORTAL_SEARCH_PAGE);
+
```
Comment (anguss00) [inline selenium/src/test/java/au/org/emii/portal/tests/step1/FacetPlatformTest.java]: and extra spaces here

**Comment by akashisama** [comment]
Annoying whitespace removed

**Pull request merged**